source/designers-guide/eslint-for-plugins/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: ESLint for plugins
+github_link: designers-guide/find-smarty-blocks/index.md
+indexed: true
+group: Frontend Guides
+subgroup: General Resources
+menu_title: ESLint for plugins
+---
+
+### Introduction
+Shopware features an automatic static code analyses tool called [ESLint](http://eslint.org/). Combined with a shared configuration called [standardjs](https://standardjs.com/) we provided a convenient way to ensure a consistent code style in the core & in our plugins.
+
+In this guide we'll talking a look on how you as a Shopware plugin developer can use the benefits of ESLint in your own plugin.
+
+### Installation
+ESLint has Node.js and NPM as an dependency you have to install beforehand. If you haven't installed them, please go ahead and install it using the provided [download methods](https://nodejs.org/en/download/current/).
+
+#### Installing the dependencies globally
+Next we're installing ESLint and it's dependencies globally on your system. You have to install them once and be ready for every other plugin you'll write.
+
+Use the following command to install ESLint, the standardjs configuration & necessary plugins:
+
+```
+npm install -g eslint eslint-config-standard eslint-plugin-import eslint-plugin-node eslint-plugin-promise eslint-plugin-standard
+```
+
+#### Installating the dependencies per plugin
+If you're not feeling comfortable installing the dependencies globally, you can also create a `package.json` file in your plugin directory (e.g. `SwagBundle` for example). You can create this file using the command `npm init`.
```

Comment:
(e.g. `SwagBundle` for example) is double 

e.g. actually means for example :wink: